docs/supported-technologies.asciidoc

```diff
@@ -80,6 +80,8 @@ The Node.js agent will automatically instrument the following modules to give yo
 |https://www.npmjs.com/package/elasticsearch-js[elasticsearch-js] |>=8.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/graphql[graphql] |>=0.7.0 <15.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/handlebars[handlebars] |* |Will instrument compile and render calls
+|https://www.npmjs.com/package/jade[jade] |>=0.5.6 |Will instrument compile and render calls
+|https://www.npmjs.com/package/pug[pug] |'0.1.0 || >=2.0.1' |Will instrument compile and render calls
```

Comment:
We don't exclude buggy module versions from our supported version range, that could quickly become a very long list and doesn't really add any informant information. If users were to install these they would not work anyway - with or without this agent... Some modules have many buggy versions 😬 We just exclude them from our test-matrix as they obviously can't be tested. This makes this list a lot easier to read.

```suggestion
|https://www.npmjs.com/package/pug[pug] |'>=0.1.0' |Will instrument compile and render calls
```

Reply:
OK, got it! I missed the review before. :)